instagram/src/CommentSection/Comment.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Comment = (props) => {
+    return (
+        <div>
```

Comment:
You can also use fragments now `<>` instead of a div to wrap child components in React